src/core/aws-messages.adb

```diff
@@ -103,6 +103,13 @@ package body AWS.Messages is
    S508_Message : aliased constant String := "Loop Detected";
    S510_Message : aliased constant String := "Not Extended";
    S511_Message : aliased constant String := "Network Authentication Required";
+   S520_Message : aliased constant String := "Unknown Error";
+   S521_Message : aliased constant String := "Web Server Is Down";
+   S522_Message : aliased constant String := "Connection Timed Out";
+   S523_Message : aliased constant String := "Origin Is Unreachable ";
```

Comment:
Ending space before " should be removed.

Reply:
Fixed!